pymc/distributions/multivariate.py

```diff
@@ -2102,18 +2101,18 @@ class CAR(Continuous):
 
     Parameters
     ----------
-    mu: array
+    mu : tensor_like of float
         Real-valued mean vector
-    W: Numpy matrix
+    W : ndarray of int
         Symmetric adjacency matrix of 1s and 0s indicating
         adjacency between elements.
```

Comment:
I am not familiar at all with the distribution. I suspect the "matrix" was used to indicate "at least 2d". So it might be something like `mu : (..., M) tensor_like ...` and `W : (..., M, M) tensor_like ...`. IIRC, @junpenglao wrote an example notebook about CAR, so he probably is able to help solve this.

Note: similarly to the other cases, it might be `M` and `(M, M)` for now with plans to be updated to taking batched inputs

Reply:
Several of our multivariate distributions only work with matrix inputs, so unless someone tests it it should remain a part of the docs. 

2D tensor_like or something.

One of this year GSOC projects is focused on allowing multivariate distributions to broadcast arbitrarily, but this one will probably not be worked on.

Reply:
The goal is to document the actual accepted input, the main issue is I have no idea what that is. Not when it comes to shape nor when it comes to accepted types.

The current `numpy matrix` is somewere between ambiguous, wrong and concerning. One might think it means the only accepted inputs are those of type [numpy.matrix](https://numpy.org/doc/stable/reference/generated/numpy.matrix.html) a type not recommended to use that might even get removed at some point, it might also mean 2d numpy arrays which also seems odd, it probably accepts aesara tensors too, and being a symetric matrix it isn't really any 2d array/tensor, hence the `(M, M) tensor_like` (or `(..., M, M) tensor_like` if it already supports batched inputs).

If in addition there are shape restrictions between multiple parameters, they should also be documented. Seeing as this looks similar to a multivariate normal, where there are these restrictions, I suggested updating `mu` too together with the changes to `W`.